docs/infrastructure.md

```diff
@@ -70,22 +70,17 @@ Note that your Stardog license file must be in the directory specified by `NB_GR
     shell variable of the same name set, 
     the shell variable will take precedence over the configuration
     of `.env`!
-    Either unset the local variable or always export the `.env` file first.
+    In this case, make sure to `unset` the local variable first.
 
     For more information, see [Docker's environment variable precedence](https://docs.docker.com/compose/environment-variables/envvars-precedence/).
 
-To export all the variables defined in your `.env` file in one step, run the following:
-```bash
-export $(cat .env | xargs)
```

Comment:
I think keeping this in the docs is still helpful, what do you think?

Reply:
Aha. This change was actually based on [feedback from Thomas](https://github.com/neurobagel/documentation/issues/67) who is redeploying Neurobagel @ the Douglas, but I agree with it since in our current docker compose setup we want the .env file to be the single 'source of truth' about the environment variables (esp because the .env file is persistent, whereas shell vars are not). As we've seen, exporting the variables as an extra step is more error-prone since you might end up w/ two sets of values that are mismatched, (e.g., if you export the defaults and then decide you want to update the .env file, the exported values will still override the new ones you have set). To make things easiest for the user and easiest to troubleshoot, they should be able to use just .env for everything _when using docker compose_, which is our recommended sysadmin deployment route. Note that in the API README, we still recommend exporting the variables for alternate installation routes where it's actually needed, e.g. if deploying using Python. :)